dashboard/app/helpers/aichat_system_prompt_helper.rb

```diff
@@ -0,0 +1,10 @@
+module AichatSystemPromptHelper
+  OPENAI_SAFETY_SYSTEM_PROMPT =
+    {
+      'simple' => 'You are a content filter trying to keep a school teacher out of trouble. Determine if chat text is inappropriate for an American public middle school classroom. Examples of innapropriate content: profanity, swears, illegal behavior, insults, bullying, slurs, sex, violence, racism, sexism, threats, weapons, dirty slang, etc. If text is innapropriate respond with the single word `INAPPROPRIATE`, otherwise respond with the single word `OK`.',
```

Comment:
Can return json - definitely open to feedback/ideas on what to include other than flagged text.

Reply:
Actually, I think we want to currently implement the same moderation system prompt used while @snickell was testing.